### ISSUE TITLE
Color contrast

### ISSUE THREAD

**Issue by mgifford** [pages-themes/minimal#474]
I may have posted this in the wrong spot initially, but wanted to report this color contrast issue here too:
https://github.com/orderedlist/minimal/issues/79

The color for small doesn't meet color contrast requirements

```
a small {
    font-size: 11px;
    color: #777;
    margin-top: -0.3em;
    display: block;
}
```

It should be:
`color: #717171 `

See: https://contrast-ratio.org/#%23777-on-white
